packages/react-juce/src/components/ScrollView.ts

```diff
@@ -1,20 +1,94 @@
-import React from 'react';
+import React, { PropsWithChildren } from 'react';
 
 import invariant from 'invariant';
 
 function ScrollViewContentView(props: any) {
   return React.createElement('ScrollViewContentView', props, props.children);
 }
 
-export function ScrollView(props: any) {
+function parseScrollbarColorProp(scrollbarColorProp: string, props: any) {
+  const values = scrollbarColorProp.split(' ');
+
+  invariant(
+    values.length === 2,
+    'scrollbar-color should be a space separated string with two values: "{thumbColor} {trackColor}".'
+  );
+
+  props['scrollbar-thumb-color'] = values[0];
+  props['scrollbar-track-color'] = values[1];
+}
+
+function parseScrollbarWidthProp(scrollBarWidthProp: string | number, props: any) {
+  invariant(
+    typeof scrollBarWidthProp === 'string' || typeof scrollBarWidthProp === 'number',
+    'scrollbar-width should be a string or a number.'
+  )
+
+  if (typeof scrollBarWidthProp === 'string') {
+    switch(scrollBarWidthProp) {
+      // JUCE's look and feel V2 uses 18px as default scrollbar thickness.
+      case 'auto':
+        props['scrollbar-width'] = 18;
+        return;
+      case 'thin':
+        props['scrollbar-width'] = 9;
+        return;
+      case 'none':
+        props['overflow'] = 'hidden';
```

Comment:
This is roughly the spec of 'none'. Is enabling `scrollOnDrag` the right behaviour here?

Reply:
Yea, I like that